packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
+    text-align: center;
+    white-space: initial;
+    width: 20%;
+  }
+
+  .min-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-right: 7px;
+    text-align: left;
+    white-space: initial;
+  }
+
+  .max-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-left: 7px;
+    text-align: left;
+    white-space: initial;
+  }
+
+  .input-range {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background-color: transparent;
+    margin-left: 7px;
+    margin-right: 7px;
+    min-height: 44px;
+    min-width: 240px;
+    width: 70%;
+  }
+  
+  .input-range:disabled {
+    opacity: 30%;
+  }
+
+  .input-range::-webkit-slider-runnable-track {
+    /* Hides the slider so that custom slider can be made */
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range, #007cc3) 0%, var(--terra-slider-track-selected-range, #007cc3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) 100%);
+    border-radius: 3px;
+    height: 6px;
+    width: 6px;
+  }
+
+  .input-range::-webkit-slider-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: var(--terra-slider-thumb-background, #007cc3);
+    border-color: var(--terra-slider-thumb-border-color, #0065a3);
+    border-radius: 50%;
```

Comment:
Did the border radius of 50% come from a UX callout?  Usually the value is in rem units.

Reply:
Updated to rems.